org.apache.xml.security.test.resource.TestVectorResolver.init : r0 = "org.apache.xml.security.test.resource.TestVectorResolver" >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : r1 = "testvectors.zip"
org.apache.xml.security.test.resource.TestVectorResolver.init : r1 = "testvectors.zip" >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $z0 = <org.apache.xml.security.test.resource.TestVectorResolver: boolean alreadyInitialized>
org.apache.xml.security.test.resource.TestVectorResolver.init : $z0 = <org.apache.xml.security.test.resource.TestVectorResolver: boolean alreadyInitialized> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : if $z0 != 0
org.apache.xml.security.test.resource.TestVectorResolver.init : if $z0 != 0 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: boolean alreadyInitialized> = 1
org.apache.xml.security.test.resource.TestVectorResolver.init : if $z0 != 0 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : return
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: boolean alreadyInitialized> = 1 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r3 = new java.util.HashMap
org.apache.xml.security.test.resource.TestVectorResolver.init : $r3 = new java.util.HashMap >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r3.<java.util.HashMap: void <init>(int)>(30)
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r3.<java.util.HashMap: void <init>(int)>(30) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> = $r3
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> = $r3 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> = $r3 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> = $r3 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r4 = new java.util.zip.ZipInputStream
org.apache.xml.security.test.resource.TestVectorResolver.init : $r4 = new java.util.zip.ZipInputStream >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r4 = new java.util.zip.ZipInputStream >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r4 = new java.util.zip.ZipInputStream >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r5 = staticinvoke <java.lang.Class: java.lang.Class forName(java.lang.String)>(r0)
org.apache.xml.security.test.resource.TestVectorResolver.init : $r5 = staticinvoke <java.lang.Class: java.lang.Class forName(java.lang.String)>(r0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r5 = staticinvoke <java.lang.Class: java.lang.Class forName(java.lang.String)>(r0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r5 = staticinvoke <java.lang.Class: java.lang.Class forName(java.lang.String)>(r0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r6 = virtualinvoke $r5.<java.lang.Class: java.io.InputStream getResourceAsStream(java.lang.String)>(r1)
org.apache.xml.security.test.resource.TestVectorResolver.init : $r6 = virtualinvoke $r5.<java.lang.Class: java.io.InputStream getResourceAsStream(java.lang.String)>(r1) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r6 = virtualinvoke $r5.<java.lang.Class: java.io.InputStream getResourceAsStream(java.lang.String)>(r1) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r6 = virtualinvoke $r5.<java.lang.Class: java.io.InputStream getResourceAsStream(java.lang.String)>(r1) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r4.<java.util.zip.ZipInputStream: void <init>(java.io.InputStream)>($r6)
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r4.<java.util.zip.ZipInputStream: void <init>(java.io.InputStream)>($r6) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r4.<java.util.zip.ZipInputStream: void <init>(java.io.InputStream)>($r6) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r4.<java.util.zip.ZipInputStream: void <init>(java.io.InputStream)>($r6) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> = $r4
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> = $r4 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> = $r4 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> = $r4 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : n0 = null
org.apache.xml.security.test.resource.TestVectorResolver.init : n0 = null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : n0 = null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : n0 = null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>
org.apache.xml.security.test.resource.TestVectorResolver.init : $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r8 = virtualinvoke $r7.<java.util.zip.ZipInputStream: java.util.zip.ZipEntry getNextEntry()>()
org.apache.xml.security.test.resource.TestVectorResolver.init : $r8 = virtualinvoke $r7.<java.util.zip.ZipInputStream: java.util.zip.ZipEntry getNextEntry()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r8 = virtualinvoke $r7.<java.util.zip.ZipInputStream: java.util.zip.ZipEntry getNextEntry()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r8 = virtualinvoke $r7.<java.util.zip.ZipInputStream: java.util.zip.ZipEntry getNextEntry()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : r9 = $r8
org.apache.xml.security.test.resource.TestVectorResolver.init : r9 = $r8 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : r9 = $r8 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : r9 = $r8 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : if $r8 == null
org.apache.xml.security.test.resource.TestVectorResolver.init : if $r8 == null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $z1 = virtualinvoke r9.<java.util.zip.ZipEntry: boolean isDirectory()>()
org.apache.xml.security.test.resource.TestVectorResolver.init : if $r8 == null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= return]
org.apache.xml.security.test.resource.TestVectorResolver.init : if $r8 == null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : if $r8 == null >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : r30 = $r29 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : return
org.apache.xml.security.test.resource.TestVectorResolver.init : r28 = $r27 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= return]
org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= return] >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : return
org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= return] >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : return
org.apache.xml.security.test.resource.TestVectorResolver.init : $z1 = virtualinvoke r9.<java.util.zip.ZipEntry: boolean isDirectory()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $z1 = virtualinvoke r9.<java.util.zip.ZipEntry: boolean isDirectory()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $z1 = virtualinvoke r9.<java.util.zip.ZipEntry: boolean isDirectory()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : if $z1 != 0
org.apache.xml.security.test.resource.TestVectorResolver.init : if $z1 != 0 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r10 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>
org.apache.xml.security.test.resource.TestVectorResolver.init : if $z1 != 0 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>
org.apache.xml.security.test.resource.TestVectorResolver.init : if $z1 != 0 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : if $z1 != 0 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r10 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r10 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r10 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : r2 = staticinvoke <org.apache.xml.security.utils.JavaUtils: byte[] getBytesFromStream(java.io.InputStream)>($r10)
org.apache.xml.security.test.resource.TestVectorResolver.init : r2 = staticinvoke <org.apache.xml.security.utils.JavaUtils: byte[] getBytesFromStream(java.io.InputStream)>($r10) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : r2 = staticinvoke <org.apache.xml.security.utils.JavaUtils: byte[] getBytesFromStream(java.io.InputStream)>($r10) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : r2 = staticinvoke <org.apache.xml.security.utils.JavaUtils: byte[] getBytesFromStream(java.io.InputStream)>($r10) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r11 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors>
org.apache.xml.security.test.resource.TestVectorResolver.init : $r11 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r11 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r11 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.HashMap vectors> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r12 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>()
org.apache.xml.security.test.resource.TestVectorResolver.init : $r12 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r12 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r12 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r11.<java.util.HashMap: java.lang.Object put(java.lang.Object,java.lang.Object)>($r12, r2)
org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r11.<java.util.HashMap: java.lang.Object put(java.lang.Object,java.lang.Object)>($r12, r2) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r11.<java.util.HashMap: java.lang.Object put(java.lang.Object,java.lang.Object)>($r12, r2) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r11.<java.util.HashMap: java.lang.Object put(java.lang.Object,java.lang.Object)>($r12, r2) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r14 = <org.apache.xml.security.test.resource.TestVectorResolver: org.apache.log4j.Category cat>
org.apache.xml.security.test.resource.TestVectorResolver.init : $r14 = <org.apache.xml.security.test.resource.TestVectorResolver: org.apache.log4j.Category cat> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r14 = <org.apache.xml.security.test.resource.TestVectorResolver: org.apache.log4j.Category cat> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r14 = <org.apache.xml.security.test.resource.TestVectorResolver: org.apache.log4j.Category cat> >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r15 = new java.lang.StringBuilder
org.apache.xml.security.test.resource.TestVectorResolver.init : $r15 = new java.lang.StringBuilder >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r15 = new java.lang.StringBuilder >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r15 = new java.lang.StringBuilder >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r15.<java.lang.StringBuilder: void <init>()>()
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r15.<java.lang.StringBuilder: void <init>()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r15.<java.lang.StringBuilder: void <init>()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : specialinvoke $r15.<java.lang.StringBuilder: void <init>()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r16 = virtualinvoke $r15.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Contents of ")
org.apache.xml.security.test.resource.TestVectorResolver.init : $r16 = virtualinvoke $r15.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Contents of ") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r16 = virtualinvoke $r15.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Contents of ") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r16 = virtualinvoke $r15.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Contents of ") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r17 = virtualinvoke $r16.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r0)
org.apache.xml.security.test.resource.TestVectorResolver.init : $r17 = virtualinvoke $r16.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r17 = virtualinvoke $r16.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r17 = virtualinvoke $r16.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r18 = virtualinvoke $r17.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("/")
org.apache.xml.security.test.resource.TestVectorResolver.init : $r18 = virtualinvoke $r17.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("/") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r18 = virtualinvoke $r17.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("/") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r18 = virtualinvoke $r17.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("/") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r19 = virtualinvoke $r18.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r1)
org.apache.xml.security.test.resource.TestVectorResolver.init : $r19 = virtualinvoke $r18.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r1) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r19 = virtualinvoke $r18.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r1) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r19 = virtualinvoke $r18.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(r1) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r20 = virtualinvoke $r19.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("#")
org.apache.xml.security.test.resource.TestVectorResolver.init : $r20 = virtualinvoke $r19.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("#") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r20 = virtualinvoke $r19.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("#") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r20 = virtualinvoke $r19.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("#") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r21 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>()
org.apache.xml.security.test.resource.TestVectorResolver.init : $r21 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r21 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r21 = virtualinvoke r9.<java.util.zip.ZipEntry: java.lang.String getName()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r22 = virtualinvoke $r20.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>($r21)
org.apache.xml.security.test.resource.TestVectorResolver.init : $r22 = virtualinvoke $r20.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>($r21) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r22 = virtualinvoke $r20.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>($r21) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r22 = virtualinvoke $r20.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>($r21) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" ")
org.apache.xml.security.test.resource.TestVectorResolver.init : $r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" ") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" ") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r23 = virtualinvoke $r22.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" ") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $i0 = lengthof r2
org.apache.xml.security.test.resource.TestVectorResolver.init : $i0 = lengthof r2 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $i0 = lengthof r2 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $i0 = lengthof r2 >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0)
org.apache.xml.security.test.resource.TestVectorResolver.init : $r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r24 = virtualinvoke $r23.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r25 = virtualinvoke $r24.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" bytes")
org.apache.xml.security.test.resource.TestVectorResolver.init : $r25 = virtualinvoke $r24.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" bytes") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r25 = virtualinvoke $r24.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" bytes") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r25 = virtualinvoke $r24.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>(" bytes") >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r26 = virtualinvoke $r25.<java.lang.StringBuilder: java.lang.String toString()>()
org.apache.xml.security.test.resource.TestVectorResolver.init : $r26 = virtualinvoke $r25.<java.lang.StringBuilder: java.lang.String toString()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r26 = virtualinvoke $r25.<java.lang.StringBuilder: java.lang.String toString()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : $r26 = virtualinvoke $r25.<java.lang.StringBuilder: java.lang.String toString()>() >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r14.<org.apache.log4j.Category: void debug(java.lang.Object)>($r26)
org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r14.<org.apache.log4j.Category: void debug(java.lang.Object)>($r26) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r14.<org.apache.log4j.Category: void debug(java.lang.Object)>($r26) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : virtualinvoke $r14.<org.apache.log4j.Category: void debug(java.lang.Object)>($r26) >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>]
org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>] >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r27 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>] >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r29 := @caughtexception
org.apache.xml.security.test.resource.TestVectorResolver.init : goto [?= $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>] >>>>>>>> org.apache.xml.security.test.resource.TestVectorResolver.init : $r7 = <org.apache.xml.security.test.resource.TestVectorResolver: java.util.zip.ZipInputStream zis>
